src/main/resources/reference.conf

```
@@ -37,4 +37,31 @@ hmrc-timeout-dialog {
   defaultCountdownInSeconds = 120
 }
 
+pta-account-menu {
+  pertax-frontend {
+    host = "http://localhost:9232"
+    host = ${?platform.frontend.host}
+  }
+
+  accountHome = {
+    host = ${?pta-account-menu.pertax-frontend.host}
+    path = "/personal-account"
+  }
+
+  messages = {
+    host = ${?pta-account-menu.pertax-frontend.host}
+    path = "/personal-account/messages"
+  }
+
+  checkProgress = {
+    host = ${?pta-account-menu.pertax-frontend.host}
+    path = "/track"
+  }
+
+  yourProfile = {
+    host = ${?pta-account-menu.pertax-frontend.host}
+    path = "/personal-account/your-profile"
+  }
+}
```

What about this? I've taken the tracking-frontend port from [pertax-frontend](https://github.com/hmrc/pertax-frontend/blob/main/conf/application.conf#L240-L242) and I changed the menu keys to be kebab-case delimited rather than camelCase to make it consistent within our config block, could go either way - not sure what the convention is here but figure we should be consistent
```suggestion
pta-account-menu {
  pertax-frontend {
    host = "http://localhost:9232"
    host = ${?platform.frontend.host}
  }

  tracking-frontend {
    host = "http://localhost:9100"
    host = ${?platform.frontend.host}
  }

  account-home = {
    host = ${pta-account-menu.pertax-frontend.host}
    path = "/personal-account"
  }

  messages = {
    host = ${pta-account-menu.pertax-frontend.host}
    path = "/personal-account/messages"
  }

  check-progress = {
    host = ${pta-account-menu.tracking-frontend.host}
    path = "/track"
  }

  your-profile = {
    host = ${pta-account-menu.pertax-frontend.host}
    path = "/personal-account/your-profile"
  }
}
```

---

if you do this, then you can also change the getOptionals below and just have get string, because the fallback stuff happens in the reference conf, not duplicated in the code